### PR TITLE
Fix wazuh

### PR DESCRIPTION
**Pull request by amitnarwal-sec**
We have deployed wazuh SIEM using the fix-wazuh branch in our production environment. Let's get these changes merged into the master. This will lend us on wazuh v3.13.4-1

### CODE REVIEW

File: roles/elastic-stack/ansible-elasticsearch/tasks/main.yml [hunk 1]
```diff
@@ -128,6 +128,7 @@
     validate_certs: no
     status_code: 200,401
     return_content: yes
+    force_basic_auth: yes
```
Comment (ghost) [inline roles/elastic-stack/ansible-elasticsearch/tasks/main.yml:131]: is this our change?

Comment (amitnarwal-sec) [reply roles/elastic-stack/ansible-elasticsearch/tasks/main.yml:131]: Yes, we have this in our change.

**Review by ghost — APPROVED** [review]
I've really got no idea what's our code and what's wazuh's, It makes this impossible to review. My strong preference would be update the version by merging the chosen version AND a review containing just our diff.

**Comment by amitnarwal-sec** [comment]
@frank-switchdin Just for a little clarity on code. So as per our code, we were on v3.13.0, and in our prod environment, we were on v3.13.2. So I made one branch called fix-wazuh to get changes for version v3.13.4 (the latest and last minor upgrade) to make everything on the same page. After merging these changes we will have all the latest changes to our master for v3.13.4. In this PR master is our copy and fix-wazuh is changes we got from the wazuh code base.

**Review by clay-switchdin — APPROVED** [review]

**Pull request merged**